src/NServiceBus.Core/IConfiguredEndpoint.cs

```diff
@@ -0,0 +1,25 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using ObjectBuilder;
+
+    /// <summary>
+    /// Represents an endpoint in the start-up phase.
+    /// </summary>
+    public interface IConfiguredEndpoint
+    {
+        /// <summary>
+        /// Starts the endpoint and returns a reference to it.
+        /// </summary>
+        /// <param name="builder">The adapter for the containers resolve API.</param>
```

Comment:
```suggestion
        /// <param name="builder">The adapter for the container's resolve API.</param>
```

Or maybe "container resolve API"